exercises/complex-numbers/src/ComplexNumbers.hs

```diff
@@ -0,0 +1,41 @@
+module ComplexNumbers
+(Complex(..),
+ conjugate,
+ abs,
+ real,
+ imaginary,
+ mul,
+ add,
+ sub,
+ div) where
+
+import Prelude hiding (div, abs) -- you have to redifine this two functions
```

Comment:
I don't think the comment is necessary since we did not use it in any other exercise that has `hiding`, but if it is, it would have to be spelled "redefine these two functions"